graphql/index.d.ts

```diff
@@ -2781,7 +2794,40 @@ declare module "graphql/utilities/typeFromAST" {
 }
 
 declare module "graphql/utilities/TypeInfo" {
-    class TypeInfo { }
+    import { GraphQLSchema } from 'graphql/type/schema';
+    import {
+        GraphQLOutputType,
+        GraphQLCompositeType,
+        GraphQLInputType,
+        GraphQLField,
+        GraphQLArgument,
+    } from 'graphql/type/definition';
+    import { GraphQLDirective } from 'graphql/type/directives';
+    import { ASTNode } from 'graphql/language/ast';
+
+    /**
+     * TypeInfo is a utility class which, given a GraphQL schema, can keep track
+     * of the current field and type definitions at any point in a GraphQL document
+     * AST during a recursive descent by calling `enter(node)` and `leave(node)`.
+     */
+    class TypeInfo {
+        constructor(
+            schema: GraphQLSchema,
+            // NOTE: this experimental optional second parameter is only needed in order
+            // to support non-spec-compliant codebases. You should never need to use it.
+            // It may disappear in the future.
+            getFieldDefFn: Function
```

Comment:
It should be `getFieldDefFn: getFieldDef`.

Solution

getFieldDef:
```
export interface getFieldDef {
  (
    schema: GraphQLSchema,
    parentType: GraphQLType,
    fieldNode: FieldNode
  ): GraphQLField
}
```

Although getFieldDef is a function and doesn't export, I think it needs to build a type in module.